doc/en/api/1.0.0/usergroup.yaml

```diff
@@ -101,15 +99,14 @@ paths:
             ID:
               description: ID of the newly created user
               type: string
-  /usergroup/users/{user}:
+  /usergroup/user/{user}:
     get:
       operationId: userDefaultGet
       tags:
        - "UserGroup"
       summary: Query a user
-      description: Query a user in the default user/group service.
+      description: Not Implemented.  Query a user in the default user/group service.
```

Comment:
I'm not sure about adding "Not implemented" in the description... wouldn't it be better to just remove it?
The reason I'm saying this, is that the default rendering does not show the description, and one could think the method is actually available, e.g.:

![image](https://user-images.githubusercontent.com/325433/181724021-ebfb62f8-bb53-494a-9f34-11468aacd41c.png)

If the entry is not there instead, it's probably clearer that it's not available.

Reply:
Yes, I agree @aaime.  I just didn't want to delete what seemed like an important method "Query a user" without flagging the deficiency in some way.  Commit added.